GherkinSyncTool/GherkinSyncTool/Synchronizers/TestRailSynchronizer/Content/SectionSynchronizer.cs

```diff
@@ -61,11 +70,10 @@ public ulong GetOrCreateSectionId(string path)
         {
```

Comment:
Let's specify what path expected?
`/// <param name="relativePath">Relative path to .feature file</param>
`
`public ulong GetOrCreateSectionId(string relativePath) `